ecc/Dockerfile.boilerplate-ecc

```diff
@@ -36,3 +36,5 @@ copy ecc/${CC_NAME} ${CC_PATH}/chaincode
 copy ecc_enclave/_build/lib/libsgxcc.so ${CC_LIB_PATH}/
 
 WORKDIR ${CC_PATH}
+EXPOSE 9999
```

Comment:
As with the fixed `fpccc` name, it seems to me that even this fixed port imposes a single-chaincode deployment.
Though this can be solved by overriding it at docker-run time.

Taken from a different perspective, as we have a boilerplate docker image that exposes 9999, then the later kubernetes deployment is forced (??) to start the service using that port. This raises the question (at least for me) whether kubernetes only publishes by default the ports exposed by a container, or it can actually define the port to be published.

Reply:
I added a build-arg for the CAAS port to remove the hardcoded port.

My understanding is that you define the port that our app container exposes via `spec.containers.ports`.